File: Python/fizzbuzz_oneline.py
// Python3 FizzBuzz - Oneliner solution
// Author: @aqc1

print(''.join([ "FizzBuzz\n" if num % 0x0F == 0 else "Fizz\n" if num % 0b011 == 0 else "Buzz\n" if num % 0o05 == 0 else f"{str(num)}\n" for num in range(1, 101) ]))

File: JAVA/fizzbuzzoneline.java
public class fizzbuzzoneline {
    public static void main(String[] args) {
        for (int i = 1; i <= 100; i++)
            System.out.println((i % 15 == 0) ? "FizzBuzz" : (i % 5 == 0) ? "Buzz" : (i % 3 == 0) ? "Fizz" : i);

    }
}
